ops/terraform/env/mgmt/ecr.tf

```diff
@@ -37,3 +42,94 @@ resource "aws_ecr_repository" "bfd" {
     scan_on_push = true
   }
 }
+
+#Enforce a general policy for all repository in the registry
```

Comment:
I think it's unnecessary to comment that the `aws_ecr_registry_policy` is a policy for the ecr registry.
```suggestion
```